### ISSUE TITLE
Code Contribution: [Easy] [Operator Development] glu

### ISSUE THREAD

**Issue by StrongSpoon** [FlagOpen/FlagGems#391]
## Description 任务介绍
Develop forward function for glu operator.
开发glu算子的前向功能。
## Requirements 任务要求
**Interface 接口**
`glu(Tensor self, int dim=-1) -> Tensor`
**Function reference 功能参考**
https://pytorch.org/docs/stable/generated/torch.nn.functional.glu.html#torch.nn.functional.glu
**Implementation reference 实现参考**
https://github.com/FlagOpen/FlagGems/blob/master/src/flag_gems/ops/gelu.py

The operator should support all optional arguments defined in the interface. 
算子应支持接口中定义的所有参数选项。
Please provide both accuracy test and performance test code.
请同时提供实现正确性测试与性能测试代码。
## DDL 提交时间
Please submit a Pull Request within 1 week after accepting the assignment.
请于接取任务后一周内提交PR。

**Comment by tianxiao-baai** [comment]
认领

**Comment by meinie0826** [comment]
May I ask if this is currently in an idle state or if someone is working on it? If no one is doing it, I have raised a PR, and if you have time, please give me some advice. https://github.com/FlagOpen/FlagGems/pull/630

**Comment by Miner-debug** [comment]
> May I ask if this is currently in an idle state or if someone is working on it? If no one is doing it, I have raised a PR, and if you have time, please give me some advice. [#630](https://github.com/FlagOpen/FlagGems/pull/630)

hi there, no one is working on this task. You can have a try
